README.md

```diff
@@ -7,3 +7,53 @@
 [![Try it on gitpod](https://img.shields.io/badge/try-on%20gitpod-brightgreen.svg)](https://gitpod.io/#https://github.com/PrismarineJS/prismarine-rng)
 
 Collection of utilities to work with minecraft RNG.
+
+## API
+
+## LCG
+?tbd
+
+## Random
+?tbd
+
+## crackPlayerSeed
+### crackPlayerSeed.crack(bits)
+Returns the decrypted player seed
+* `Returns` - Seed number
+* `bits` - Encrypted seed string?
```

Comment:
tbh, this is function is a bit useless without an example. I need to add it.
bits expect and array of 20 values recorded from throwing objects on the ground and measuring their velocity.

Reply:
Added some example on how to use it https://github.com/PrismarineJS/prismarine-rng/blob/master/examples/enchant.js
Would be nice to add a notice in the doc that this is highly experimental and only a library to help building seed cracking tool, not a tool itself. I think for someone to be able to truely using it, he must know the basics of seed cracking already.